es-models/gene_expression/gene_expression.mapping.yaml

```diff
@@ -0,0 +1,31 @@
+_source:
+  excludes:
+  - genes.*
+properties:
+  age_at_diagnosis:
+    type: long
+  case_id:
+    type: keyword
+  days_to_death:
+    type: long
+  ethnicity:
+    type: keyword
+  gender:
+    type: keyword
+  genes:
+    properties:
+      expression_value:
+        type: float
+      gene_id:
+        type: keyword
+      hgnc_symbol:
```

Comment:
I think this field is just called `symbol` in gene_centric, and mutation indexer maps it from the other name in the MAF. We might want to use the same name... probably not a big deal...

Reply:
...or maybe it's `hgnc`? Now I'm not actually sure.

Reply:
...okay, I looked more. I think `symbol` in gene_centric is the HGNC symbol, and `hgnc` is the "HGNC ID," which is something else. So `symbol` is probably a more consistent name.

Reply:
I'm following agreement between @ZhenyuZ and @philtom-ctds from a slack conversation they had a couple of days ago

Reply:
discussed offline and decided to use `symbol`